C/Clingcon/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Clingcon"
+version = v"5.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/potassco/clingcon.git", "586b23ceadff349051dd0a58467679a3758199cb")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/clingcon
+mkdir build && cd build
+cmake -G Ninja \
+    -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DPYCLINGCON_ENABLE=OFF \
+    -DCMAKE_CXX_FLAGS="-std=c++17" \
+    -DCMAKE_INSTALL_LIBDIR=lib \
+    ..
+cmake --build . --target install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("clingcon", :clingcon),
+    LibraryProduct("libclingcon",:libclingcon)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency("Clingo_jll")
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version = v"8")
```

Comment:
Can we require Julia v1.6?  It'll make it easier to add new platforms in the future
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version = v"8", julia_compat="1.6")
```

Reply:
I don't see why not, I have no use for <1.6 myself. Should I be doing that by default now? For new builders?

Reply:
That'd be good, yes, see https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/1047 and https://github.com/JuliaPackaging/Yggdrasil/issues/2763